nipype/pipeline/engine/utils.py

```diff
@@ -1021,7 +1022,10 @@ def export_graph(graph_in, base_dir=None, show=False, use_execgraph=False,
                                suffix='.dot',
                                use_ext=False,
                                newpath=base_dir)
-    nx.write_dot(pklgraph, outfname)
+    if LooseVersion(nx.__version__) < LooseVersion('1.11'):                           
+        nx.write_dot(pklgraph, outfname)
+    else:
+        nx.drawing.nx_pydot.write_dot(pklgraph, outfname)
```

Comment:
It looks like the change just removed the direct import inside nx: https://github.com/networkx/networkx/pull/1929